crates/sui-types/src/gas.rs

```diff
@@ -51,7 +51,7 @@ pub mod checked {
 
     /// Version aware enum for gas status.
     #[enum_dispatch(SuiGasStatusAPI)]
-    #[derive(Debug)]
+    #[derive(Debug, Serialize, Deserialize)]
```

Comment:
I don't fully follow while all types need `Deserialize`, I don't see anywhere where data is deserialized into these structs?